### ISSUE TITLE
Performance Using T5

### ISSUE THREAD

**Issue by ramarasty** [PlusLabNLP/TempGen#3]
Hello.

Thank you so much for the great work. I tried to test with t5-base for REE, but my results are lower than expected. I disabled copying to avoid bart copy:

```
Test
================= PerpInd =================
P: 48.91%,  R: 32.61%, F1: 39.13%

================= PerpOrg =================
P: 57.81%,  R: 45.12%, F1: 50.68%

================= Target =================
P: 54.43%,  R: 31.62%, F1: 40.00%

================= Victim =================
P: 45.24%,  R: 40.00%, F1: 42.46%

================= Weapon =================
P: 62.00%,  R: 51.67%, F1: 56.36%

***************** micro_avg *****************
P: 52.57%,  R: 37.96%, F1: 44.09%
```

The results are not terrible, but I think there might be some setting that is negatively affecting performance on T5. I played around with weight decay and lr, but this is the best I got. Do you have any thoughts on what the issue might be?
[Config file](https://drive.google.com/file/d/1lnnWyGSb9RQF6gvizw_kZ-nvSrCJ5XVN/view?usp=sharing) in case you want to have a look.

**Comment by khuangaf** [comment]
Hi, thanks for testing out our framework on T5! Indeed fine-tuning with T5 can be difficult, as it usually uses very different hyper-parameters than other large LMs. Some tips include using the AdaFactor optimizer instead of AdamW and increasing the learning rate. Here are more tips discussed in the HiggingFace forum: https://discuss.huggingface.co/t/t5-finetuning-tips/684/2.
